### DISCUSSION TITLE
How do I mass delete snapshots?

### DISCUSSION THREAD

**Discussion by UntouchedWagons** [longhorn/longhorn#5931] [Q&A]
For some reason some of my volumes have an excessive number of snapshots. Longhorn reports there are 249 snapshots over the 100 snapshot threshold. I don't know why there's so many snapshots. I don't remember what I set the retain on the snapshot job but I know it wasn't triple digits. How do I bulk delete snapshots? I'm using Longhorn 1.4.1.

Thanks

**Comment by UntouchedWagons** [comment]
Also how do I find out why Longhorn isn't making backups of some of my volumes? I made a backup schedule that applies to all volumes and some do have backups but others don't.

**Comment by innobead** [comment]
cc @ChanYiLin

**Comment by innobead** [comment]
https://www.reddit.com/r/kubernetes/comments/13iikpq/comment/jkit5vx/?utm_source=share&utm_medium=web2x&context=3

**Comment by UntouchedWagons** [comment]
[supportbundle_7752520e-592d-4abc-ab2e-d4b9accd2942_2023-05-17T20-11-49Z.zip](https://github.com/longhorn/longhorn/files/11508268/supportbundle_7752520e-592d-4abc-ab2e-d4b9accd2942_2023-05-17T20-11-49Z.zip)

Reuploaded the bundle since the original had my name in it about 500 times.
